src/simple-pasta-cream-sauce.md

```diff
@@ -0,0 +1,25 @@
+# Simple Creamy Pasta Sauce
+
+- ⏲️  Prep time: 2 min
+- 🍳  Cook time: 3 mins
+- 🍽️  Servings: 1
+
+## Ingredients
+
+- Dairy liquid cream (10%+ fat recommended)
+- Flour (Wheat flour recommended)
+
+## Directions
+
+ 1. Add 1 teaspoon of flour into a saucepan.
+ 2. Pour ~60ml(2fl oz|1/4 cup) of cream into a saucepan.
+ 3. Stir and turn on maximum heat.
+ 4. Keep stirring until it becomes thick.  
+ If the sauce is too liquid, add more flour, if the sauce is too solid (like a uncooked pancake), add more cream.
```

Comment:
Is this newline in the list intentional? I think it looks better if it's either 1) same line as 4. or 2) the line starts at 5.